### ISSUE TITLE
Any tips on running this on CI

### ISSUE THREAD

**Issue by tijs** [UrbanCompass/Mussel#16]
I'm having some issues running this on our Bitrise builds so I'm hoping someone may have run into this issue as well or has some insight that might get me on my way again..

I've added some push notifications tests for our app, they run fine locally (very cool project so thanks!), but on bitrise builds the same tests fail with this less than helpful error message:

```
Failed to get matching snapshot: Lost connection to the application (pid 25406). (Underlying Error: Couldn't communicate with a helper application. Try your operation again. If that fails, quit and relaunch the application and try again. The connection to service on pid 0 created from an endpoint was invalidated.)
```

I assume the helper application referred to here is the XCTest helper app. But since all our other tests run fine I don't really understand why this particular test would fail.

We use Xcode 12.4 both locally and on Bitrise.

**Comment by luispadron** [comment]
@tijs Hey! We're working on updating this to work better with CI. We'll even be releasing a dedicated Bitrise step soon enough. Thanks for your patience here

**Comment by tijs** [comment]
@luispadron oh that great news! we've been running it locally for now so we can at least run it sometimes. But yeah on CI would be much better of-course.

**Comment by luispadron** [comment]
Hi again @tijs, we have just released version `1.0.0` of Mussel as well as the Bitrise integration: https://www.bitrise.io/integrations/steps/mussel. Let us know if this helps you use Mussel in CI! 

Checkout the updated README for more info
